src/lib/gundb/UserStorageClass.js

```diff
@@ -1422,13 +1422,13 @@ export class UserStorage {
     ])
 
     //Issue with gun delete()
-    // let profileDelete = await this.gunuser
-    //   .delete()
-    //   .then(r => ({ profile: 'ok' }))
-    //   .catch(e => ({
-    //     profile: 'failed'
-    //   }))
-    logger.debug('deleteAccount', { deleteResults })
+    let userDelete = await this.gunuser
```

Comment:
So, if this fails invoker function gets `deleteResults` anyway?

Reply:
refactored. yeah it made no sense to await on it not with the others

Reply:
sounds good, but this result shouldn't be part of the response? so invoker can tell if the user was deleted from gun?

Reply:
it is looked at the new code